Codeinwp/neve
# [Not Ready Yet] Onboarding moved to composer module

### Summary
Onboarding moved to the composer module. It is not in codeinwp/ti-onboarding repo.
The demo sites are still in the theme folder, in the **onboarding** directory.

### Will affect visual aspect of the product

NO

### Screenshots 

### Test instructions


- Clone the theme
- ``` composer install ```
- Try to import something within the about page -> sites library
- Try to open the onboarding from the button within the welcome notice and check 'Skip this step', it should be a blue button with the previous quoted text.


Closes #.
